### DISCUSSION TITLE
Reordering Records - Add default sort value when new record is created.

### DISCUSSION THREAD

**Discussion by isaacdarcilla** [filamentphp/filament#11491] [Ideas]
https://filamentphp.com/docs/3.x/tables/advanced#reordering-records

Add default sort value when new record is created. So, if there are existing sorted records in the table, automatically add a sort value in a newly created record.

**Comment by leandrocfe** [comment]
[mutateFormDataBeforeCreate](https://filamentphp.com/docs/3.x/panels/resources/creating-records#customizing-data-before-saving) ?
